### PR TITLE
RDM-6151-directory structure changed

### PR DESCRIPTION
**Pull request by akdasari**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: ccd-user-profile-importer/values.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+userProfileUrl: http://ccd-user-profile-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/users
+s2sUrl: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+users: 
+jurisdictions:
+waitHosts: ""
+waitHostsTimeout: 300
+microservice: ccd_definition
+verbose: "false"
+userProfileDatabaseHost: "{{ .Release.Name }}-postgresql"
+userProfileDatabasePort: "5432"
```
Comment (smathangi) [inline ccd-user-profile-importer/values.yaml]: Actually POST "/user-profile/users" endpoint creates jurisdiction record if it's not there already. So, you don't need all these database config, create-jurisdiction.sh etc.

Comment (akdasari) [reply ccd-user-profile-importer/values.yaml]: it doesn't work, without this configuration, see the way cmc uses this importer
https://github.com/hmcts/chart-cmc/blob/master/cmc/values.yaml#L45

Comment (smathangi) [reply ccd-user-profile-importer/values.yaml]: They need to remove that unnecessary config once upgrade to latest ccd chart.

File: ccd-user-profile-importer/templates/user-profile-importer.yaml [hunk 2]
```diff
@@ -6,44 +7,33 @@ metadata:
     {{- ( include "ccdupi.labels" . ) | indent 4 }}
   annotations:
     helm.sh/hook: post-install,post-upgrade
-    helm.sh/hook-weight: "5"
-    helm.sh/hook-delete-policy: before-hook-creation  # Move to hook-succeeded after first charts are done 
+    helm.sh/hook-weight: "-5"
```
Comment (adusumillipraveen) [inline ccd-user-profile-importer/templates/user-profile-importer.yaml]: Why is the weight changed from old ccd chart?

File: ccd-user-profile-importer/values.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+userProfileUrl: http://ccd-user-profile-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+s2sUrl: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+users: 
+waitHosts: ""
+waitHostsTimeout: 300
+microservice: ccd_definition
+verbose: "false"
+
+global:
+  environment: aat
```
Comment (adusumillipraveen) [inline ccd-user-profile-importer/values.yaml]: May be not good to default here.

File: ci-values.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+dummyConfigInCI: true   #This Flag enables dummy-config for pipleine build only
+users:
+  - adasari123.hmcts.claimant+1@gmail.com|CMC|MoneyClaimCase|open
+verbose: true
+ 
+global:
+  subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
```
Comment (adusumillipraveen) [inline ci-values.yaml]: Are these unused?

File: Dockerfile [hunk 5]
```diff
@@ -10,4 +10,4 @@ RUN ["chmod", "+x", "/wait"]
 
 RUN chmod +x /scripts/*.sh
 
-CMD "/wait" && "/scripts/create-jurisdiction.sh" && "/scripts/create-users.sh"
```
Comment (adusumillipraveen) [inline Dockerfile:13]: Is create judistriction not needed? Have seen other related changes removed

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]

File: ccd-user-profile-importer/values.yaml (hunk 1 above)
Comment (akdasari) [reply ccd-user-profile-importer/values.yaml]: removed db attributes

File: ccd-user-profile-importer/templates/user-profile-importer.yaml (hunk 2 above)
Comment (akdasari) [reply ccd-user-profile-importer/templates/user-profile-importer.yaml]: changed back to original weight

File: ci-values.yaml (hunk 4 above)
Comment (akdasari) [reply ci-values.yaml]: not needed , removed

File: Dockerfile (hunk 5 above)
Comment (akdasari) [reply Dockerfile:13]: user-profile-url end point creates jurisdictions, so no need of separate script

File: ccd-user-profile-importer/values.yaml (hunk 3 above)
Comment (akdasari) [reply ccd-user-profile-importer/values.yaml]: moved to ci-values

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**